tests/conftest.py

```diff
@@ -7,7 +7,7 @@
 import memcache
 import pytest
 
-import aiomcache
```

Comment:
This looks a touch odd to me. I'd rather add a `from aiomcache.helpers import ...` line.

Reply:
6 half dozen ;). It's fairly standard.  This way is nice because you get both aiomcache and .helpers imported in one line. went ahead and changed it because I don't really care one way or the other